src/coreclr/scripts/superpmi_asmdiffs_checked_release_setup.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
+#
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+#
+# Title               : superpmi_asmdiffs_checked_release_setup.py
+#
+# Notes:
+#
+# Script to setup directory structure required to perform SuperPMI asmdiffs checked release in CI.
+# It creates `correlation_payload_directory` with `base` and `diff` directories
+# that contains clrjit*.dll. 
+#
+################################################################################
+################################################################################
+
+import argparse
+import logging
+import os
+
+from coreclr_arguments import *
+from jitutil import copy_directory, set_pipeline_variable, run_command, TempDir, download_files
+
+parser = argparse.ArgumentParser(description="description")
+
+parser.add_argument("-arch", help="Architecture")
+parser.add_argument("-source_directory", help="path to the directory of the dotnet/runtime source tree")
+parser.add_argument("-product_directory", help="path to the directory containing built binaries (e.g., <source_directory>/artifacts/bin/coreclr/windows.x64.Checked)")
```

Comment:
```suggestion
parser.add_argument("-checked_directory", help="path to the directory containing checked binaries (e.g., <source_directory>/artifacts/bin/coreclr/windows.x64.Checked)")
```

If we're explicit about 'release', we should be explicit about 'checked' also.

A more general option would be to just use the words "base" and "diff", and (at least in this file) we'd assume one was the checked JIT, one the release. Or maybe both could be release. E.g.:
```
parser.add_argument("-base_directory", help="path to the directory containing baseline binaries (e.g., <source_directory>/artifacts/bin/coreclr/windows.x64.Checked)")
parser.add_argument("-diff_directory", help="path to the directory containing diff binaries (e.g., <source_directory>/artifacts/bin/coreclr/windows.x64.Release)")
```

Reply:
changed to base and diff.